SEC/Securities/SecuritiesIssuance.rdf

```diff
@@ -547,6 +615,14 @@
 		<skos:definition>indicates a period of time in which investors can commit to purchase shares (or units) to be issued</skos:definition>
 	</owl:ObjectProperty>
 	
+	<owl:ObjectProperty rdf:about="&fibo-sec-sec-iss;isIssuedInForm">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-agr-ctr;isEvidencedBy"/>
+		<rdfs:label>is issued in form</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fbc-fi-fi;Security"/>
+		<rdfs:range rdf:resource="&fibo-sec-sec-iss;SecurityForm"/>
```

Comment:
Major problem - this means that BearerForm, RegisteredForm should be Individuals not subClasses or you get punning.

Reply:
I've addressed this through a series of changes to how these are used - they were already individuals, but the restrictions didn't use them consistently.  Took a couple of passes to get this right.